conf/minion

```diff
@@ -20,6 +20,7 @@
 #proxy_port:
 #proxy_username:
 #proxy_password:
+#no_proxy: []
```

Comment:
IMHO, we should also add some documentation here so that a person doesn't have to refer to the `.rst` file to know what this does since it isn't immediately obvious.

Reply:
@mrproper I think this is the last requested change before we can approve this. Thanks!